test/unit/services/search_indexer_test.rb

```diff
@@ -36,9 +36,15 @@ def test_indexing_to_rummager
 
   def test_format_exceptions_are_not_indexed
     SearchIndexer::FORMATS_NOT_TO_INDEX.each do |format|
-      artefact = FactoryGirl.create(
-        :artefact, kind: format, content_id: "content-id",
-      )
+      if format == 'completed_transaction'
+        artefact = FactoryGirl.create(:artefact,
```

Comment:
Could you pop that :artefact on a new line?
cf. last paragraph of https://github.com/alphagov/styleguides/blob/master/ruby.md#general